### PR TITLE
You can now click on symptoms in the Pandemic to see their description and stats

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: You can now click on symptoms in the Pandemic to see their description and stats
/:cl:

Also includes fancy descriptions of symptom effects.

Pics:
snip
snip2

### CODE REVIEW

**Comment by Supermichael777** [comment]
ewww a feature

this is good though

**Comment by GunHog** [comment]
If it were a bit more condensed, you could put this stuff into a tooltip, so the user can just read what he wants and then hover over the the next.

**Comment by XDTM** [comment]
I plan on also adding threshold information in the symptom pages (as soon as i figure out how to do it without copypasta), which would make tooltips a bit too big. Maybe i can put in the description or the stats on a tooltip anyway.

**Comment by XDTM** [comment]
Added threshold information on symptoms.

snip

File: code/datums/diseases/advance/advance.dm [hunk 1]
```diff
@@ -171,6 +171,7 @@
 	var/datum/disease/advance/A = SSdisease.archive_diseases[GetDiseaseID()]
 	AssignName(A.name)
 
+
```
Comment (optimumtact) [inline code/datums/diseases/advance/advance.dm]: stop this

Comment (Cyberboss) [inline code/datums/diseases/advance/advance.dm]: whitespace creep

**Comment by XDTM** [comment]
@Cyberboss fixed conflicts

**Pull request merged**